### PR TITLE
Fixed bug #44 - incorrect function name

### PR DESCRIPTION
**Pull request by zhenyab**
@pwalsh Please, review

### CODE REVIEW

File: .idea/codeStyleSettings.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pwalsh) [inline .idea/codeStyleSettings.xml]: @zhenyab cant have this here.

Comment (zhenyab) [inline .idea/codeStyleSettings.xml]: @pwalsh strange, I have added .idea folder to gitignore, but still see it in commit :)

Removed from GIT repo

**Comment by rgrp** [comment]
@zhenyab is there any way to "compress" / rebase so we don't have the addition of the IDE folder then its removal?

**Comment by zhenyab** [comment]
@rgrp in pull request it is not a problem, because it will squash commits when merged, and history of the master will remain clean. Is this what you mean?

**Comment by pwalsh** [comment]
@zhenyab I merged. Next time, try to separate actual fixes for issues from general code cleanup.

@rgrp using squash and merge we do not get the IDE stuff in the master history.

**Pull request merged**